docs/Release.md

```diff
@@ -38,3 +37,10 @@ Release is performed by two commands
 mvn release:clean release:prepare -Prelease
 mvn release:perform -Prelease
 ```
+
+---
+>_Note_: The gpg agent should ask for the passphrase to your gpg key automatically, in case that does not happen you can try adding similar property to the `<properties>` section:   
```

Comment:
```suggestion
>_Note_: The gpg agent should ask for the passphrase to your gpg key automatically, in case that does not happen you can try to add property to the `<properties>` section:   
```